src/StateL1Null.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Null implementation for L1 state holder.
+ */
+
+namespace LCache;
+
+/**
+ * Fake/Stub state manager class.
+ *
+ * @author ndobromirov
+ */
+class StateL1Null extends StateL1Static implements StateL1Interface
```

Comment:
I'm concerned about how this splits the `PHP_INT_MAX` for `getLastAppliedEventID` away from `NullL1`. It allows incorrect behavior when someone uses this state implementation with a stateful L1. As a rule, it should be possible to combine any StateL1 with any L1 and have correct (but maybe not ideal) behavior.

Perhaps, we should just not have a `StateL1Null` and use `StateL1Static` with `NullL1` when testing.